src/views/RegionSeasonalRiskWatch/index.tsx

```diff
@@ -0,0 +1,629 @@
+} from '#utils/risk';
+import type {
+    HazardType,
+    RiskDataItem,
+    RiskMetric,
+    RiskMetricOption,
+} from '#utils/risk';
+import { useRiskRequest } from '#utils/restRequest';
+import { formatNumber, maxSafe, sumSafe } from '#utils/common';
+import {
+    COLOR_LIGHT_BLUE,
+    COLOR_LIGHT_GREY,
+    COLOR_PRIMARY_RED,
+} from '#utils/constants';
+import { type RegionOutletContext } from '#utils/outletContext';
+
+import Filters from './Filters';
+import type { FilterValue } from './Filters';
+import styles from './styles.module.css';
+
+// eslint-disable-next-line import/prefer-default-export
+export function Component() {
+    const { regionResponse } = useOutletContext<RegionOutletContext>();
+    const bbox = useMemo(
+        () => (regionResponse ? getBbox(regionResponse.bbox) : undefined),
+        [regionResponse],
+    );
+
+    const { regionId } = useParams<{ regionId: string }>();
+    const [filters, setFilters] = useState<FilterValue>({
+        months: [],
+        countries: [],
+        riskMetric: 'exposure',
+        hazardTypes: [],
+        normalizeByPopulation: false,
+        includeCopingCapacity: false,
+    });
+
+    const { response: seasonalResponse } = useRiskRequest({
+        skip: isNotDefined(regionId),
+        apiType: 'risk',
+        url: '/api/v1/seasonal/',
+        query: { region: regionId },
+    });
+
+    const { response: riskScoreResponse } = useRiskRequest({
+        skip: isNotDefined(regionId),
+        apiType: 'risk',
+        url: '/api/v1/risk-score/',
+        query: {
+            region: isDefined(regionId) ? Number(regionId) : undefined,
+            limit: 500,
+        },
+    });
+
+    const seasonalRiskData = seasonalResponse?.[0];
+
+    const mappings = useMemo(
+        () => {
+            if (!riskScoreResponse || !riskScoreResponse.results) {
+                return undefined;
+            }
+
+            const lcc = listToMap(
+                riskScoreResponse.results.map(
+                    (item) => {
+                        if (!item.country_details
+                            || isFalsyString(item.country_details.iso3)
+                            || isNotDefined(item.lcc)
+                        ) {
+                            return undefined;
+                        }
+
+                        return {
+                            ...item,
+                            lcc: item.lcc,
+                            country_details: {
+                                ...item.country_details,
+                                iso3: item.country_details.iso3,
+                            },
+                        };
+                    },
+                ).filter(isDefined),
+                (item) => item.country_details.iso3 ?? '',
+                (item) => item.lcc,
+            );
+
+            const population = listToMap(
+                riskScoreResponse.results,
+                (item) => item.country_details.iso3 ?? '',
```

Comment:
Let's filter out the list if there is not iso3

Reply:
It's a look-up map, so it shouldn't matter